uaclient/config.py

```diff
@@ -520,11 +520,15 @@ def _attached_service_status(
                 description_override = inapplicable_resources[ent.name]
             else:
                 ent_status, details = ent.user_facing_status()
+
         return {
             "name": ent.name,
             "description": ent.description,
             "entitled": contract_status.value,
             "status": ent_status.value,
+            "available": status.UserFacingAvailability.UNAVAILABLE.value
+            if ent.name in inapplicable_resources
+            else status.UserFacingAvailability.AVAILABLE.value,
```

Comment:
Since we're not actually going to display this value, can we just use a boolean here instead of the string values?

Reply:
We could, but I made that field consistent with the `unattached` status json. Since changing it there will be considered a schema change, I preferred to follow that pattern here as well.

Reply:
Ah I see what you mean now, cool.

I do think we should move towards `status` returning data, and having separate formatting functions take care of replacing the data with a user-readable strings if necessary. So in this case, that would mean returning a bool here (for both attached and unattached), and then interpreting the bool as a string over in `format_tabular`. That's just my preference though and certainly doesn't have to be in scope for this PR.

Reply:
I think this a good suggestion. I think we can keep this as is for now. As we work on the status json spec and see other similar behaviors like this, we can have a PR that makes all of those changes at once. I think this will be easier to address when we have a better understanding of all the data changes we want to perform

Reply:
Totally, sounds good to me!

Reply:
@lucasmoura @orndorffgrant also before releasing this change, we need coordinate with desktop folks RobertAncell because they currently match ["available"== "yes"](https://git.launchpad.net/ubuntu/+source/software-properties/tree/softwareproperties/gtk/utils.py#n127) to determine if is a service is applicable when unattached. If we want to release this changeset in our next SRU, we'll need to minimally [file a bug file against software-properties](https://bugs.launchpad.net/ubuntu/+source/software-properties/+filebug) and/or help provide a minor PR to fix this and wait/help with an SRU release for software-properties that would fix this behavior in order to that any service listed when unattached and lacking an "available" key in status.json can still be  determined as "available".

Reply:
Got it. I can align that with them